extensions/ray-so/src/generate-image.tsx

```diff
@@ -15,20 +13,24 @@ export default async () => {
   let selectedText;
   try {
     selectedText = await getSelectedText();
-  } catch (e) {
+  } catch {
     await showHUD(
       "❌ Screenshot generation failed. Please make sure you've selected the text you want to take a screenshot of.",
     );
     return;
   }
 
-  const base64Text = encodeURI(selectedText);
+  const base64Text = Buffer.from(selectedText, "utf8")
+    .toString("base64")
+    .replace(/\+/g, "-")
+    .replace(/\//g, "_")
+    .replace(/=/g, "");
```

Comment:
Can we use the `encodeForRayso()` here?

Reply:
Can't believe I didn't catch that one. Thanks!